app/src/App.js

```diff
@@ -6,60 +6,6 @@ import TextField from "material-ui/TextField";
 import io from "socket.io-client";
 import { MuiThemeProvider, createMuiTheme } from 'material-ui/styles';
 
-const ENDPOINT = "http://toonin-backend-54633158.us-east-1.elb.amazonaws.com:8100/";
```

Comment:
We should probably revert this file

Reply:
Yeah, the changes on this file are irrelevant. They can be discarded safely